app/src/main/java/com/group80/uoftinder/entities/User.java

```diff
@@ -0,0 +1,126 @@
+package com.group80.uoftinder.entities;
+
+import android.net.Uri;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+
+public class User {
+
+    private final String uid;
+
+    private String userType; // make this final later idk
```

Comment:
Instead of making `userType` a `String`, which may cause potential errors (like typo), I would suggest making this an enum. 

This can be achieved by the following
```java
//UserType.java
package com.group80.uoftinder.entities;

public enum UserType {
    ACADEMIC("Academic"), // assume the current flag for academic partner finding is "Academic"
    FRIEND("Friend"), // assuming the current flad for friend-finding is "Friend"
    ROMANTIC("Romantic"); // assuming the current flag Tinder ( :,) ) user is "Romantic" 

    private String userType;

    private UserType(String userType) {
        this.userType = userType;
    }

    @Override
    public String toString() {
        return this.userType;
    }
}
```
This should allow `UserType` to behave similar to `String` when being used (so everything using your getter should not need to be changed -- just remember to explicitely call the `toString` method in your getter), while constraint the options to prevent misuse. For example, consider the following:
```java
// Using `String`
user.setUserType("Academic"); // Good
user.setUserType("academic"); // Complies fine, but we know this will cause future errors
// Using `Enum`
user.setUserType(UserType.ACADEMIC); // Good
user.setUserType(UserType.academic); // Error thrown at compile time, so we immediately know this is invalid
```